module1/homework/grayscaleImages/compression.cpp

```diff
@@ -0,0 +1,50 @@
+#include "compression.hpp"
+
+compressed_bitmap compressGrayscale(const bitmap& image) {
+    compressed_bitmap compressedImage{};
+    compressedImage.reserve(height * width);
+
+    uint8_t currentGrayscale;
+    uint8_t counter;
+    uint8_t grayscale;
+
+    for (size_t i = 0; i < height; ++i) {
+        currentGrayscale = image[i][0];
```

Comment:
And when you will use range loop. You can just use .front() to get first value of each row.

Reply:
Fixed